pkg/json/v0/config/tasks.json

```
@@ -105,5 +106,78 @@
       "title": "Output",
       "type": "object"
     }
+  },
+  "TASK_JQ": {
+    "instillShortDescription": "Process JSON through a jq command",
+    "input": {
+      "description": "Source JSON and jq command",
+      "instillUIOrder": 0,
+      "properties": {
+        "jsonInput": {
+          "instillUIOrder": 0,
+          "description": "JSON string to be processed",
+          "instillAcceptFormats": [
+            "string"
+          ],
+          "instillUpstreamTypes": [
+            "value",
+            "reference",
+            "template"
+          ],
+          "instillUIMultiline": true,
+          "title": "JSON input",
+          "type": "string"
+        },
+        "jqFilter": {
+          "instillUIOrder": 1,
+          "description": "Filter, in jq syntax, that will be applied to the JSON input",
+          "instillAcceptFormats": [
+            "string"
+          ],
+          "instillUpstreamTypes": [
+            "value",
+            "reference",
+            "template"
+          ],
+          "instillUIMultiline": true,
+          "title": "JQ filter",
+          "type": "string"
+        }
+      },
+      "required": [
+        "jsonInput",
+        "jqFilter"
+      ],
+      "title": "Input",
+      "type": "object"
+    },
+    "output": {
+      "description": "Results of the jq command applied to the input JSON",
+      "instillEditOnNodeFields": [
+        "json"
+      ],
+      "instillUIOrder": 0,
+      "properties": {
+        "results": {
+          "description": "The jq command results. Depending on the filter and the input JSON the type of each element may vary.",
+          "instillEditOnNodeFields": [],
+          "instillUIOrder": 0,
+          "required": [],
+          "title": "Results",
+          "type": "array",
+          "instillFormat": "semi-structured/json",
+          "items": {
+            "instillAcceptFormats": [
+              "semi-structured/json"
+            ]
+          }
```

```suggestion
          "items": {
            "instillFormat": "semi-structured/json"
          }
```

In the output schema, we will use `instillFormat` instead of `instillAcceptFormats` since we will only have one possible data type.